geometry_msgs/Point[] blockedCells
geometry_msgs/Point[] edgeCells